corral/cmd/_test_cmd_fetch.pony

```diff
@@ -0,0 +1,102 @@
+use "files"
+use "ponytest"
+use "../bundle"
+use "../util"
+use "../vcs"
+
+actor _TestCmdFetch is TestList
+  new make() =>
+    None
+
+  fun tag tests(test: PonyTest) =>
+    test(_TestFetchEmptyDeps)
+    test(_TestFetchEmptyFile)
+    test(_TestFetchMutuallyRecursive)
+    test(_TestFetchSelfReferential)
```

Comment:
I think you should add a test for transitive dependencies. That would mean renaming the regression-120 stuff to be a more generic "transitive dependencies" test.

Reply:
will do